package.json

```diff
@@ -9,6 +10,7 @@
   "scripts": {
     "all": "yarn clean && yarn build && yarn test && yarn lint && yarn docs",
     "clean": "lerna run clean",
+    "dev": "cross-env NODE_ENV=development SERVE=1 node ./bin/build.mjs",
```

Comment:
what does it do?

Reply:
`yarn dev packages/three-vrm` to start developing on a package, shorter and faster compared to `NODE_ENV=development SERVE=1 yarn workspace @pixiv/three-vrm build`

Reply:
who specifies the `packages/three-vrm` ?

Reply:
`packages/three-vrm` is the arg (`process.argv[2]`) pass to cli. See https://github.com/pixiv/three-vrm/pull/1411/files#diff-a9c6b99e70a85f0bb0fd35f083215fec11ae9bddf6e29853d04831ce1b813ec9R14

Reply:
running `yarn dev` with out arg will error out but I don't think that case worth handling

Reply:
ah got it got it, you said `yarn dev packages/three-vrm`. looks good